src/modules/poweraccent/PowerAccent.UI/Selector.xaml

```diff
@@ -6,7 +6,7 @@
         xmlns:local="clr-namespace:PowerAccent"
         mc:Ignorable="d" SizeToContent="WidthAndHeight" ShowInTaskbar="False" ResizeMode="NoResize"
         Title="MainWindow" Height="50" Width="50" Visibility="Collapsed" WindowStyle="None">
-    <Grid>
```

Comment:
why this?

Reply:
Oops, wrong commit. That shouldn't be there.